apps/core/scenes/search/SearchContext.js

```diff
@@ -178,6 +181,18 @@ export default class SearchContextProvider extends React.Component<
     });
   };
 
+  setStateFromQueryParams = (params: Object) => {
+    const parsedParams = qs.parse(params);
```

Comment:
qs.parse takes a string, so is params an object or as string? Could you give a type to parsedParams, so that we know what we are working with.

Reply:
Done!